### PR TITLE
release(testnet): GA 2021.11.10.0

### PR DESCRIPTION
**Pull request by marvinmarnold**
**Why**
Keep up with GA releases.

**How**
Uses [new hm-miner by Mr. Bump](https://github.com/NebraLtd/hm-miner/commit/6ee720d968173f50c92a67f538bc69837dd61b9f).

**References**
Related to: https://github.com/NebraLtd/helium-miner-software/issues/203

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -34,7 +34,7 @@ services:
       - pktfwdr:/var/pktfwd
```
Comment (shawaj) [inline docker-compose.yml:34]: Need to do the version number too

Comment (shawaj) [reply docker-compose.yml:34]: Oops tagged wrong line, but lines 11 and 63

Comment (marvinmarnold) [reply docker-compose.yml:34]: Thanks

**Review by shawaj — APPROVED** [review]

**Comment by KevinWassermann94** [comment]
Blocked, Testnet units seem to be reporting offline in explorer

**Pull request merged**